libs/perun/components/src/lib/members-list/members-list.component.html

```diff
@@ -134,6 +134,15 @@
         <th *matHeaderCellDef mat-header-cell>{{'MEMBERS_LIST.LOGINS' | translate}}</th>
         <td *matCellDef="let member" mat-cell>{{member | memberLogins}}</td>
       </ng-container>
+      <ng-container matColumnDef="consentStatus">
+        <th *matHeaderCellDef mat-header-cell mat-sort-header>
+          {{'CONSENTS.STATUS' | translate}}
+        </th>
+        <td mat-cell *matCellDef="let member">
+          <perun-web-apps-consent-status
+            [consentStatus]="member.userId | consentStatusIcon: consents"></perun-web-apps-consent-status>
```

Comment:
There's no need to call the pipe here, since we already called it in the parent and assigned the consent to MemberWithConsent.